applications/hamster.md

```diff
@@ -0,0 +1,278 @@
+
+*Zhiwei Li*: Graduate student in Computer Science, Substrate pallet developer.
+
+### Team Code Repos
+
+- https://github.com/hamster-shared
+- https://github.com/hamster-shared/frontend
+- https://github.com/hamster-shared/hamster
+- https://github.com/hamster-shared/hamster-client
+- https://github.com/hamster-shared/hamster-provider
+
+
+
+- https://github.com/ltmuyuan
+- https://github.com/mohaijiang
+- https://github.com/jian-guo-s
+- https://github.com/LwwL-123
+
+## Development Status :open_book:
+
+We have completed a simple version of the project for minimal process execution, and will subsequently update the usage documentation so you can install and experience using it.
+
+The following is a list of the features that have been implemented:
+
+- Hamster Node
+
+  Computer resource register
+
+  Update computer resource unit-price
+
+  Create resource order
+
+  Provider finish order
+
+  Provider heartbeat
+
+  Provider staking amount
+
+  Provider withdraw amount
+
+  Withdraw rental amount
+
+  Withdraw fault execution
+
+  Client cancel order
+
+- Hamster Provider
+
+  Computer create
+
+  Computer destruction
+
+  P2p link listen
+
+- Hamster Client
+
+  List of paid computers
+
+  Computer link usage
+
+- Hamster Front End
+
+  Computing market
+
+  Provider shared computing
+
+  Provider lease agreement list
+
+  List of computers purchased by the customer
+
+  Customer order list
+
+- Hamster Gateway
+
+  Private IPFs network
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** ~~3 months~~ 2months
+- **Full-Time Equivalent (FTE):**  4 FTE
+- **Total Costs:** ~~$38,400~~ $25,600
+
+### Milestone 1 Example — Implement Hamster Client and Provider Modules
+
+- **Estimated duration:** 1 month
+- **FTE:**  4
+- **Costs:** $12,800
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | Apache 2.0 |
+| 0b. | Documentation | We will provide online documentation of the code and a basic tutorial that includes<br/>1. Hamster Client installation tutorial<br/>2. Hamster Client usage tutorial<br/>3. Hamster Provider installation tutorial<br/>4. Hamster Provider usage tutorial<br/>5. Hamster Chain installation tutorial<br/>6. Hamster Chain usage tutorial |
+| 0c. | Testing Guide | Core functions will be fully covered by unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 1. | Hamster Client | We will create a desktop client that will have **Market**,**My order** and **My resource** module.<br/>We will provide a desktop (windows, macos) app based on [wails](https://wails.io/) to achieve<br/>1: Market inquiry and management of purchased resources,<br/>2: The point-to-point communication between the client and the computing resource provider.<br/>3: Query historical orders<br/>4: Partial fee refund for defaulted orders<br/>Tech stack: go+wails+go-libp2p+vue.js+polkadot.js |
```

Comment:
Could you deliver a Linux version of the desktop client? Note that all software delivered as part of a grant [must not rely on closed-source software](https://github.com/w3f/Grants-Program#guidelines).

Reply:
We use Wails is an open source go-based development client framework, we currently use his V2 version is to support windows and macos, we can provide subsequent Linux version of the desktop client

Reply:
Fair enough, I see Wails v2 will eventually support Linux too. Thanks.